packages/@aws-cdk/aws-stepfunctions-tasks/lib/eks/call.ts

```diff
@@ -0,0 +1,104 @@
+import * as iam from '@aws-cdk/aws-iam';
+import * as sfn from '@aws-cdk/aws-stepfunctions';
+import { Construct } from 'constructs';
+import { integrationResourceArn, validatePatternSupported } from '../private/task-utils';
+
+/** Properties for calling a EKS endpoint with EksCall */
+export interface EksCallProps extends sfn.TaskStateBaseProps {
+
+  /** Name of the cluster */
+  readonly clusterName: string;
+
+  /** Base 64 encoded certificate data required to communicate with your cluster */
+  readonly certificateAuthority: string;
+
+  /** API endpoint to communicate with your cluster */
+  readonly endpoint: string;
+
+  /** The HTTP method ("GET", "POST", "PUT", ...) that clients use to call this method */
+  readonly httpMethod: MethodType;
+
+  /** Path of cluster */
+  readonly path: string;
+
+  /**
+   * Path of cluster
+   * @default - no query parameters
+   */
+  readonly queryParameters?: { [key: string]: string[] };
+
+  /**
+   * request body of the http method
```

Comment:
"Request body part of HTTP request"

Reply:
Addressed in new pull request https://github.com/aws/aws-cdk/pull/12779